pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[tool.mypy]
```

Comment:
can we keep the nice comments? at least some makes sense to me to keep

Reply:
My wabbits!!